src/v/model/transform.h

```diff
@@ -146,10 +154,20 @@ struct transform_metadata
           environment,
           uuid,
           source_ptr,
-          offset_options);
+          offset_options,
+          paused);
     }
 };
 
+struct transform_metadata_patch {
```

Comment:
Would be good to document this struct overall (i.e. where is it used?)

Reply:
done